web/help/editor.md

```diff
@@ -0,0 +1,32 @@
+Editor Shortcuts
+================
+
+    | `Shortcut`                   | `Action`                     |
+    |------------------------------|------------------------------|
+    | Ctrl + Enter                 | Run the program              |
+    | Ctrl + Space / Shift + Space | Autocomplete                 |
```

Comment:
Just a nit, but this terse notation will confuse students.  I'd rather see:

    Ctrl + Space, or Shift + Space

Same with other keys below.

Reply:
Done.